### PR TITLE
[feat]:Added PlayList Management

### PR DESCRIPTION
**Pull request by ultronjs**
-Feature Added
1.)User can make custom Playlist
2.)User can save a single video into multiple playlist
3.)Delete a video from the playlist
4.)Delete the whole playlist

### CODE REVIEW

**Comment by hussiantabha** [comment]
I like how you named your variables and destructured most of the code. Good job, Keep it up

File: src/components/PlayListCard/PlayListCard.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import "./PlayListCard.css"
+import {
+  MdOutlineDeleteOutline,
+  MdOutlineArrowForwardIos,
+} from "react-icons/md";
+import { usePlayLists } from '../../context/playlistContext'
+import { useNavigate } from "react-router-dom";
+
+function PlayListCard(props) {
+  const navigate =useNavigate()
```
Comment (sushil-mod) [inline src/components/PlayListCard/PlayListCard.jsx:12]: this is correct, also you destructure props beforehand (just a suggestion)

File: src/components/PlayListModal/PlayListModal.jsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    removeVideoInPlayList,
+  } = usePlayLists();
+  const initialCreatePlayListObj = {
+    _id: uuid(),
+    title: "",
+    description: "",
+    videos: [],
+  };
+  const [createPlayList,setCreatePlayList] = useState(initialCreatePlayListObj)
+  console.log(createPlayList)
```
Comment (sushil-mod) [inline src/components/PlayListModal/PlayListModal.jsx:30]: remove log if not in future use

File: src/components/Toast.jsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import "../index.css"
 
 function Toast({type,msg}) {
```
Comment (sushil-mod) [inline src/components/Toast.jsx:4]: Nice. I liked you made your own toast variant

**Review by sushil-mod — APPROVED** [review]
few suggestions. Rest code is well written

**Comment by ultronjs** [comment]
Merging after 2 approvals

**Pull request merged**